tests/benchmark/spaces_benchmarks/bm_spaces_class.h

```diff
@@ -15,20 +15,20 @@ class BM_VecSimSpaces : public benchmark::Fixture {
     std::mt19937 rng;
     size_t dim;
     DATA_TYPE *v1, *v2;
-    spaces::Arch_Optimization opt;
 
 public:
     BM_VecSimSpaces();
     ~BM_VecSimSpaces() = default;
 
     void SetUp(const ::benchmark::State &state);
     void TearDown(const ::benchmark::State &state);
+
+    // Specific architecture optimization flags that are supported on this machine,
+    // to be initialized in every executable that is running this benchmarks.
+    static spaces::Arch_Optimization opt;
```

Comment:
is this is use?

Reply:
It has slightly changed due to the removal of `Arch_Optimization`, but yes - for storing the optimizations in the class and testing against it (as comment says)

Reply:
I mean that it is only used once. you don't really need to store it in the class,

Reply:
Not sure what do you mean, let's discuss it